docs/source/markdown/options/security-opt.md

```diff
@@ -20,6 +20,8 @@ Note: Labeling can be disabled for all <<|pods/>>containers by setting label=fal
 
 - **mask**=_/path/1:/path/2_: The paths to mask separated by a colon. A masked path cannot be accessed inside the container<<s within the pod|>>.
 
+- **nested**: Allows SELinux modifications within the container. Containers are allowed to modify SELinux labels on files and processes, as long as SELinux policy allows. Without **nested**, containers view SELinux as disabled, even when it is enabled on the host. Containers are prevented from setting any labels.
```

Comment:
I don't like `nested` as name, both for the option but also the struct filed name in the ContainerConfig and specgen. It is just so generic, I have no idea that this is related to selinux without reading the comments.

Why not parse it as `label=nested`?